nubis/puppet/files/confd/templates/traefik.toml.tmpl

```diff
@@ -103,6 +103,24 @@ main = "dynamicua-origin.mozilla.org"
 [[acme.domains]]
 main = "start.mozilla.org"
 
+[[acme.domains]]
+main = "start.mozilla.com"
+sans = [
+  "start-prod.mozilla.com"
```

Comment:
Does this one comes for free with line 122 "*.start-prod"?

Reply:
I'm going to add it like this for now and risk having duplicates. I might run into issues requesting wildcard certificates without the dns-01 challenge type.